INSTALL.md

```diff
@@ -157,8 +157,33 @@ On other operating systems, you will need to figure out equivalent commands
 to install the required dependencies.
 
 Note that several of the packages bundled with GAP have additional
-prerequisites.
-
+prerequisites. Here is an incomplete list of GAP packages and their requirements
+(excluding GMP, which many packages require):
+
+- 4ti2Interface
+  - Debian/Ubuntu: 4ti2
+  - Homebrew: (not currently available)
+- alnuth
+  - Debian/Ubuntu: pari-gp
+  - Homebrew: pari
+- browse
+  - Debian/Ubuntu: libncurses5-dev
```

Comment:
better just `libncurses-dev`? (in fact `libncurses6-dev` seems to be used on my machine)

Reply:
Thanks for the pointer. I copied what I had added a long time ago to the CI setup, I simply was not aware of this. I've adjusted this PR and also the CI workflow to use `libncurses-dev`